includes/forms/class-wp-job-manager-form-submit-job.php

```diff
@@ -407,6 +407,22 @@ protected function validate_fields( $values ) {
 						}
 					}
 				}
+				if ( 'file' === $field['type'] ) {
+					if ( is_array( $values[ $group_key ][ $key ] ) ) {
+						$check_value = array_filter( $values[ $group_key ][ $key ] );
+					} else {
+						$check_value = array_filter( [ $values[ $group_key ][ $key ] ] );
+					}
+					if ( ! empty( $check_value ) ) {
+						foreach ( $check_value as $file_url ) {
```

Comment:
We could probably merge this check with the one above it to avoid code duplication. For example:
```
foreach ( $check_value as $file_url ) {
	if ( ! empty( $field['allowed_mime_types'] ) ) {
		// do mime checks here
	}
	// do the check that you added here.
}
```

Reply:
Good!
I took the opportunity to refactor also the other validation block related to the file.

https://github.com/Automattic/WP-Job-Manager/pull/1989/commits/422f88882a24dcc064aeb534ccf2551c163337a8